segmentstore/server/src/main/java/io/pravega/segmentstore/server/store/ServiceConfig.java

```diff
@@ -272,6 +275,12 @@
     @Getter
     private final boolean enableTls;
 
+    /**
+     * Tls Protocol Version
+     */
+    @Getter
+    private final String[] tlsProtocolVersion;
```

Comment:
Why not just keep a `TlsProtocolVersion` instance here?

Reply:
TlsProtocolVersion is just a utility class created for parsing and validating the string passed to it.